src/main/java/com/cburch/logisim/fpga/gui/FPGAClockPanel.java

```diff
@@ -163,9 +163,9 @@ public double GetTickfrequency() {
       i++;
     }
     return Double.parseDouble(number.toString()) * switch (extention) {
-      case 'K' -> 1000;
```

Comment:
What about using `D` suffix instead of `.0`? That's would be least tempting to be removed.

Reply:
Done